client/src/components/add-username-modal/add-username-modal.scss

```diff
@@ -0,0 +1,64 @@
+@import '../../scss/variables';
+@import '../../scss/colors';
+@import '../../scss/mixins';
+@import '../../scss/atoms/button';
+
+@import '../comment/comment.scss';
+
+.username-modal {
+  @include z-level(3);
+  display: flex;
+  flex-flow: column;
+  justify-content: space-around;
+  position: fixed;
+  right: $baseline;
+  bottom: $baseline;
+  width: 35vw;
+  height: 15vh;
+  border: solid $palette-light 1px;
+  border-radius: $border-radius-default;
+  background-color: $palette-lighter;
+  cursor: default;
+  &:focus {
+    outline: none;
+    box-shadow: 0 2px 10px $palette-shadow-default;
+  }
+}
+
+.add-button {
+  @include button();
+  align-self: flex-end;
+  margin-top: $baseline;
+  padding: $baseline $spacer;
+  background: $palette-accent;
+  color: $palette-lighter;
+}
+
+.modal-content {
+  align-items: center;
+  display: flex;
+  flex-flow: column;
+  flex: 1;
+  justify-content: center;
+  padding: 1em 2em;
+  input[type="text"] {
+    @extend .comment;
+    display: block;
+    min-width: 100%;
+    border: none;
+  }
+}
+
+.username-overlay {
```

Comment:
not sure if this should be lifted to higher abstraction later, as @sofiatengvall uses similar stuff.. @tobbe-j

Reply:
Could probably be left as is in this commit, and then lifted/moved in feature-intro branch where the same is used

Reply:
yes